src/components/ContentPreview/ContentPreview.js

```diff
@@ -269,8 +269,10 @@ class ContentPreview extends PureComponent<Props, State> {
      *
      * @return {void}
      */
+    /* eslint-disable no-unused-vars */
     componentDidUpdate(prevProps: Props, prevState: State): void {
```

Comment:
Here was the prior implementation without the need of componentDidUpdate / shouldLoadPreview
https://github.com/box/box-ui-elements/blob/00b9d1c624cab69644f96a6955cf9de87c9f3297/src/components/ContentPreview/ContentPreview.js#L378

Unsure if we can revert back to that?

Reply:
Basically what happened before is that loadPreview() had a this.preview guard in it. In other words if this.preview was not undefined, it was assumed there was a preview open and no need to load preview.

Then only time this.preview was destroyed, when navigation happened.

Reply:
I guess it was also destroyed in cWRP
https://github.com/box/box-ui-elements/blob/00b9d1c624cab69644f96a6955cf9de87c9f3297/src/components/ContentPreview/ContentPreview.js#L171